pkg/config/config.go

```diff
@@ -181,46 +173,29 @@ func StringInList(s string, list []string) bool {
 	return false
 }
 
-func (c *MicroshiftConfig) ReadFromConfigFile() error {
-	if len(c.ConfigFile) == 0 {
-		return nil
-	}
-
-	contents, err := os.ReadFile(c.ConfigFile)
+// Note: add a configDir parameter here because of unit test requiring custom
+// local directory
+func (c *MicroshiftConfig) ReadFromConfigFile(configDir string) error {
```

Comment:
Is the argument the full path to the file or just the directory name? It looks like it's used as a filename on the next line, but it's named like it's a directory.